### PR TITLE
Add a submodule-sync workflow

### PR DESCRIPTION
**Pull request by borkweb**

### CODE REVIEW

File: .github/workflows/submodule-sync.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        working-directory: ./tut
+        with:
+          envkey_GITHUB_USER: tr1b0t
+          envkey_GITHUB_OAUTH_TOKEN: ${{ secrets.GH_BOT_TOKEN }}
+          file_name: .env
+
+      - name: Synchronize submodules
+        working-directory: ./tut
+        run: |
+          ./tut submodule-sync
```
Comment (sc0ttkclark) [inline .github/workflows/submodule-sync.yml:69]: I think this should only happen for the current branch. We should update tut to restrict to the branch it's currently on from the common workflow

**Review by sc0ttkclark — COMMENTED** [review]
Looks great! One change might be even better/safe

File: .github/workflows/submodule-sync.yml (hunk 1 above)
Comment (borkweb) [reply .github/workflows/submodule-sync.yml]: Good call! The command doesn't support that yet. BUT...this action now works, so I'll go off and update the command to allow us to specify a specific branch :man_dancing:

Comment (sc0ttkclark) [reply .github/workflows/submodule-sync.yml]: Heck yeah!!

**Comment by borkweb** [comment]
Now that this is working swimmingly with a specific branch, I'll close this out and open a real PR

**Pull request closed without merging**